Sources/AppKitNavigation/Internal/ToOptionalUnit.swift

```diff
@@ -0,0 +1,12 @@
+#if canImport(AppKit) && !targetEnvironment(macCatalyst)
```

Comment:
This isn't used yet, so let's remove it from the PR. When it is needed it can move to SwiftNavigation and use the `package` modifier, as well.